tests/bluetooth/controller/common/defaults_cmake.txt

```diff
@@ -0,0 +1,37 @@
+#
+# Common include directories and source files for bluetooth unit tests
+#
+
+include_directories(
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/include
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/common/include
+        $ENV{ZEPHYR_BASE}/include/bluetooth
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/util
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/include
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw
+)
+
+#
+# we test functionality of ull_conn, which is included directly in the test file
+# so we do not list it here
+#
+FILE(GLOB ll_sw_sources
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/util/mem.c
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/util/memq.c
+        $ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/ll_sw/ull_tx_queue.c
+)
+
+FILE(GLOB mock_sources
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ll.c
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull.c
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull_master.c
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ticker.c
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/mayfly.c
```

Comment:
These files do not exist yet, so maybe remove them?

Reply:
Removed